tests/test_offline_policy_evaluator.py

```
@@ -473,3 +469,9 @@ def test_initialization_when_xgboost_not_available(
                 propensity_score_feature=propensity_score_feature,
             )
             assert evaluator is not None
+
+
+def test_safe_cv_raises_on_single_sample_class(labels=np.array([0, 0, 0, 0, 1])) -> None:
```

Ruff (0.15.9)

[warning] Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)